### DISCUSSION TITLE
I'm not sure what to do, game was working just fine and bam, doesn't work anymore.

### DISCUSSION THREAD

**Discussion by hauntedopera** [FabricMC/fabric-api#2887] [Player support] [answered]
Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'oωo' (owo) 0.8.2+1.19, mod 'oωo' (owo) 0.8.5+1.19, mod 'oωo-sentinel' (owo-sentinel) 0.8.0+1.19, mod 'oωo-sentinel' (owo-sentinel) 0.8.2+1.19, mod 'oωo-sentinel' (owo-sentinel) 0.9.1+1.19 and mod 'oωo-sentinel' (owo-sentinel) 0.9.3+1.19 with mod 'oωo' (owo), version 0.9.3 or later.
Unmet dependency listing:
	 - Mod 'Mythic Metals' (mythicmetals) 0.17.0+1.19.2 requires version 0.9.3 or later of mod 'oωo' (owo), which can't be loaded due to other constraints!
	 - Mod 'Mythic Metals Decorations' (mythicmetals_decorations) 0.5.2+1.19.2 requires version 0.9.3 or later of mod 'oωo' (owo), which can't be loaded due to other constraints!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

**Comment by PlainsVillager** [comment, marked as answer]
you need a mod

**Reply by hauntedopera** [reply]
If id ask, do you know what mod exactly? Usually the game tells what you need on the launcher but im puzzled on this one

**Reply by hauntedopera** [reply]
Nvm i read it more and now i feel dumb

**Reply by hauntedopera** [reply]
ugh, nvm now its complaining about projectile damage. I dont even know what that is
